client/services/code_evaluators/PrereqCheckDispatcherServiceSpec.js

```diff
@@ -0,0 +1,50 @@
+// Copyright 2017 The TIE Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit tests for the PrereqCheckDispatcherServiceSpec.
+ */
+describe('PrereqCheckDispatcherService', function() {
+  var PrereqCheckDispatcherService;
+  var LANGUAGE_PYTHON = 'python';
+
+  beforeEach(module('tie'));
+  beforeEach(inject(function($injector) {
+    PrereqCheckDispatcherService = $injector.get(
+      'PrereqCheckDispatcherService');
+  }));
+
+  describe('checkCode', function() {
+    it('correctly calls PythonPrereqCheckService when the language is python',
+      function() {
+        var code = [
+          'def myFunction(arg):',
+          '\treturn result',
```

Comment:
Just to check -- in some tests we use 4 spaces and in other tests we use \t. What is it in practice, and can we stick to that throughout? I seem to remember we standardized it in the frontend (so that if the user hits Tab it gets converted to 4 spaces, or the other way round).